code/datums/components/tippable.dm

```diff
@@ -209,3 +223,19 @@
 	else
 		tipped_mob.transform = turn(tipped_mob.transform, -180)
 		REMOVE_TRAIT(tipped_mob, TRAIT_IMMOBILIZED, TIPPED_OVER)
+
+/datum/component/tippable/proc/accept_roleplay(mob/living/user, datum/emote/emote)
+	SIGNAL_HANDLER
+
+	if (!is_tipped)
+		return
+	if (roleplayed)
+		return
+	if (!is_type_in_list(emote, roleplay_emotes))
+		return
+	var/time_left = timeleft(self_untip_timer)
+	deltimer(self_untip_timer)
+	self_untip_timer = addtimer(CALLBACK(src, .proc/right_self, user), time_left / 2, TIMER_UNIQUE | TIMER_STOPPABLE)
```

Comment:
```suggestion
	self_untip_timer = addtimer(CALLBACK(src, .proc/right_self, user), time_left * 0.8, TIMER_UNIQUE | TIMER_STOPPABLE)
```
not sure about this change, if we have it at all it should be less drastic an advantage. Either way we prefer multiplying by a decimal

Reply:
sure, i changed it to .75